pkgs/development/interpreters/python/cpython/2.7/boot.nix

```diff
@@ -26,7 +26,7 @@ stdenv.mkDerivation rec {
   LDFLAGS = optionalString (!stdenv.isDarwin) "-lgcc_s";
   NIX_CFLAGS_COMPILE = optionalString stdenv.isDarwin "-msse2";
 
-  buildInputs = optionals stdenv.isDarwin [ CF configd ];
```

Comment:
What were these for? I think the other interpreters still have these.

Reply:
Not sure about configd but that's probably optional since that was already disabled. As for CF, the darwin bootstrap-tools include CoreFoundation so that's used while building the stdenv itself.

Reply:
Please do it in a separate commit, but within this PR.

Reply:
No, we can't make packages from the final set depend on bootstrap tools.

Reply:
Okay, it might be clearer to leave these in but just null them out in stdenv/darwin/default.nix

Reply:
I am still a little confused on why this is needed. Packages don't usually depend on CF or configd. Doesn't the stdenv provide a version that non-boot cpython can pick up?